package.json

```
@@ -33,7 +33,7 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "start": "react-scripts start",
+    "start": "set PORT=3000 && react-scripts start",
```

Check if this works:

```suggestion
    "start": "PORT=3000 react-scripts start",
```

---

Unfortunately no. I think its gonna work only on Mac. Maybe should left only "start": "react-scripts start" to prevent error for different OS?